src/app/organizations/state/organizations.query.ts

```diff
@@ -50,4 +54,20 @@ export class OrganizationsQuery extends Query<OrganizationsState> {
     }
     return null;
   }
+
+  filterAndSortOrganizations(organizations: Array<Organization>, searchName: string, sortBy: string): Array<Organization> | null {
+    if (organizations) {
```

Comment:
@coverbeck  Yeah for sure, I definitely agree on changing the parameter type to `Array | something`, the thing is this function I wrote is very similar to one that's already there in this file, and the latter has `| null` instead of `[]`. I'm wondering if we should keep the `null`